pkg/universal-cnf/config/removeClientInterface_test.go

```diff
@@ -0,0 +1,243 @@
+package config
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cisco-app-networking/nsm-nse/pkg/universal-cnf/config/mocks"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connectioncontext"
+	"go.ligato.io/vpp-agent/v3/proto/ligato/vpp"
+	interfaces "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/interfaces"
```

Comment:
sort imports

Reply:
Hi Ana, just want to clarify: I believe the imports are already sorted in alphabetical order and are organized into groups. Could you specify how you would like them to be sorted?

Reply:
Hi, I saw this comment in one PR and I think it clarifies the situation: 
```
Imports should be placed in 3 groups:

- Standard library

- Third-party packages

- First-party packages from outside repositories (e.g. github.com/org/path/to/import)

```